minimal-notebook/Dockerfile

```diff
@@ -10,11 +10,11 @@ USER root
 # Install all OS dependencies for fully functional notebook server
 RUN apt-get update && apt-get install -yq --no-install-recommends \
     build-essential \
+    # Emacs installation is deprecated
     emacs-nox \
     vim-tiny \
     git \
     inkscape \
-    jed \
```

Comment:
https://github.com/jupyter/docker-stacks/pull/121#issuecomment-183843888 talks about why `jed` was proposed and added. Do we gain much by removing it?

Reply:
@parente No we do not gain much removing it. I propose not mixing editor topic discussed #1174 with this update so I will rollback changes made on editors (jed removal and emacs deprecation) and merge the update if you agree.